common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,604 @@
+#	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+#	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+#	stack = 75
+#	ai_quality = { value = @cultural_maa_extra_ai_score }
+#	icon = shatari
+#}
+
+rangari = {
+	type = archers
+	
+	damage = 34
+	toughness = 23
+	pursuit = 15
+	screen = 20
+	
+	terrain_bonus = {
+		hills = { damage = 20 toughness = 10 pursuit = 10 screen = 10 }
+		forest = { damage = 20 toughness = 20 pursuit = 15 screen = 15 }
+	}
+
+	counters = {
+		skirmishers = 1
+	}
+
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = rangari
+}
+#Demon
+infernal = { 
+	type = massive_infantry
+	
+	damage = 135
+	toughness = 128
+	pursuit = 0
+	screen = 10
+	
+	terrain_bonus = {
+		drylands = { damage = 20 toughness = 20 pursuit = 20 }
+	}
+
+	counters = {
+		skimishers = 2
+		archers = 2
+	}
+
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = infernal
+}
+infernal_machine = {
+	type = siege_weapon
+	fights_in_main_phase = no
+	
+	damage = 0
+	toughness = 12
+	
+	siege_tier = 4
+	siege_value = 1
+	
+	buy_cost = { gold = cost_115_recruitment_cost }
+	low_maintenance_cost = { gold = cost_115_low_maint_cost }
+	high_maintenance_cost = { gold = cost_115_high_maint_cost }
+	
+	stack = 10
+	icon = infernal_machine
+}
+#Dragon
+drakonid = {
+	type = massive_infantry
+	
+	damage = 135
+	toughness = 142
+	pursuit = 5
+	screen = 15
+
+	terrain_bonus = {
+		plains = { damage = 20 }
+		wetlands = { damage = -10 }
+		jungle = { damage = -20 }
+	}
+
+	counters = {
+		skirmishers = 2
+		archers = 1
+	}
+	
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = drakonid
+}
+
+##Nerubian
+arachnathid = { 
+	type = skirmishers
+	
+	damage = 14
+	toughness = 15
+	pursuit = 12
+	screen = 10
+	
+	terrain_bonus = {
+		taiga = { damage = 5 toughness = 3 pursuit = 5 screen = 5 }
+	}
+	
+	winter_bonus = {
+		mild_winter = { damage = 2 toughness = 2 pursuit = 2 screen = 2 }
+		normal_winter = { damage = 3 toughness = 3 pursuit = 3 screen = 3 }
+		harsh_winter = { damage = 4 toughness = 4 pursuit = 4 screen = 4 }
+	}
+
+	counters = {
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_55_recruitment_cost }
+	low_maintenance_cost = { gold = cost_55_low_maint_cost }
+	high_maintenance_cost = { gold = cost_55_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = culture_ai_weight_arachnathid }
+	icon = arachnathid
```

Comment:
👍

Reply:
I think to keep them this size, for now, if @zumbak04 revisits Northrend, and decides otherwise, we will go from there.  Btw not even swarm maa has the same stats as ghouls so they would both need a good look.  Nothing to change I will resolve the conversation.